crates/dbs-miniball/README.md

```diff
@@ -1,21 +1,528 @@
+  downloaded or built from sources. See [this section][#kernel] for examples on
+  how to acquire these packages using scripts from this repo.
+
+  Root privileges are needed to manage mountpoints.
+
+#### Putting It All Together
+
+Once all the prerequisites are met, the Miniball can be run either
+directly through `cargo`, passing on its specific
+[command line arguments](../README.md#cli-reference), or after building it with
+`cargo build`.
+
+```wrap
+cargo run --release --            \
+    --memory size_mib=1024        \
+    --kernel path=${KERNEL_PATH}  \
+    --vcpu num=1
+```
+
+```wrap
+cargo build --release
+target/release/vmm-reference      \
+    --memory size_mib=1024        \
+    --kernel path=${KERNEL_PATH}  \
+    --vcpu num=1
+```
 
 ## Platform Supported
-[Host OS? CPU type? guest version? rust version ? many more]
+
+### Host OS & hypervisor
+
+`rust-vmm` is ultimately meant to be usable across a wide range of host OSes
+and hypervisors, with several of the crates already supporting Windows.
+Long-term, the Miniball should support the intersection of all its crates’
+supported platforms. Currently, this intersection resolves into Linux hosts
+and the KVM hypervisor. The first iteration of the Miniball supports
+only this configuration, returning errors when users attempt to run it on
+something else.
+
+### CPU
+
+Long term, the Miniball will run on `x86_64` and `aarch64` platforms; to
+begin with, only Intel `x86_64` CPUs are supported, as there is no support
+for a PIO serial console on `aarch64`, and the virtio interfaces are at an
+early stage.
+
+### Rust features
+
+The Miniball will be compiled with all its crates’ features (at least,
+while they’re compatible with each other), and will be able to showcase usage
+with all of them in turn. Code-wise, there is only one `vmm` crate, which
+activates all its dependencies’ features through its TOML. The integration
+tests, documentation and examples will demonstrate the various features’
+functionalities. Examples of such features are: driving the event manager from
+the same thread or a new one, loading ELF and `bzImage` kernels.
+
+To exemplify this: the same Miniball binary can boot an ELF or a `bzImage`
+kernel, with no need to recompile it. Users control what it will boot, through
+the specified parameters (the user supplies the kernel image).
+
+### Rust version
+
+Rust 1.60.0
+
+### Toolchain
+
+The Miniball will support both `glibc` and `musl libc` (toolchains:
+`x86_64-unknown-linux-gnu`, `x86_64-unknown-linux-musl`) with `glibc` being the
+default due to `x86_64-unknown-linux-gnu` being
+[Tier 1 supported](https://doc.rust-lang.org/nightly/rustc/platform-support.html#tier-1)
+by Rust. Future extensions to `aarch64` support will introduce the
+`aarch64-unknown-linux-gnu` and `aarch64-unknown-linux-musl` toolchains,
+defaulting (probably) to `aarch64-unknown-linux-gnu` on ARM, because it's also
+*Tier 1 supported* since Rust 1.49.
 
```

Comment:
If we are inspired by rust-vmm's vmm-reference repo, please add an acknowledge part for their work:)

Reply:
Updated it.